### ISSUE TITLE
Settings: content languages list doesn't scroll on android

### ISSUE THREAD

**Issue by ryvwel** [bluesky-social/social-app#1325]
**Describe the bug**

Clicking "Content languages" button in settings i can't scroll languages list until i check or uncheck some language. It rarely works normal, but mostly not. Works fine in web. 

**To Reproduce**

1. Open settings
2. Open "Content languages"
3. We're here

**Expected behavior**
I can scroll languages list and click "Done" immediately

**Screenshots**
![Screenshot_2023-08-29-11-42-26-764_xyz blueskyweb app](https://github.com/bluesky-social/social-app/assets/111886772/7ac4c061-c123-4597-9d3f-951adf8ff12b)
(No "Done" button, i can't scroll)

**Details**

 - Platform: Android 
 - Platform version: 10
 - App version: 1.48.0

**Additional context**

Also, English should be on top, like in web app

**Comment by pfrazee** [comment]
@ansh not assigning this but more evidence we need to replace bottom-sheet

**Comment by ansh** [comment]
Yeah we definitely need to replace bottom sheet. It's behavior on Android is just too unpredictable. I would be inclined to use a `presentation: modal` React Navigation screen for this. Will be a nice modal on iOS with swipe to close functionality like we have now, and on Android it will render as a normal screen. Most importantly, it will be stable on both platforms. @pfrazee 

We can also work on our own bottom sheet component, but to be honest I feel like this language selector should just be it's own screen anyway.
